docs/core/testing/unit-testing-with-nunit.md

```diff
@@ -3,6 +3,8 @@ title: Unit testing C# with NUnit and .NET Core
 description: Learn unit test concepts in C# and .NET Core through an interactive experience building a sample solution step-by-step using dotnet test and NUnit.
 author: rprouse
 ms.date: 08/31/2018
+dev_langs: 
+  - "csharp"
```

Comment:
Do you want to make sure all files have `dev_langs` field? I noticed that some more are missing the value.

Reply:
If there's only one language in use, that field is not necessary, I guess.

Reply:
Yes, if it's only one language then it's not necessary.

Reply:
Correct, the field is not needed when there's only one language. We only added when we need to make the language switch appear. I'd actually remove it here.

Reply:
Okay, @mairaw. I'm going to remove it from the files where it exists :)